aioshumway/__init__.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper around shumway (https://github.com/spotify/shumway) that
+uses asyncio features to make all IO operations non-blocking and compatible
+with the async/await syntax.
+"""
+
+__author__ = 'Matt Obarzanek'
```

Comment:
I didn't know whether I should put my name or somebody else's since this is only a wrapper. Lemme know if you think I should put someone else's name + email (I wish we could put our squad details in here).